packages/a/azahar/package.yml

```diff
@@ -0,0 +1,78 @@
+# yaml-language-server: $schema=/usr/share/ypkg/schema/schema.json
+name       : azahar
+version    : '2123.3'
+release    : 1
+source     :
+    - https://github.com/azahar-emu/azahar/releases/download/2123.3/azahar-unified-source-2123.3.tar.xz : e0d321ffda7c0e372c67bfbddc92e997a29c93deae93ef790c2b38a8506fb6a9
+homepage   : https://azahar-emu.org/
+license    : GPL-2.0-or-later
+component  : games.emulator
+summary    : An open-source 3DS emulator project based on Citra
+description: |
+    Azahar is an open-source 3DS emulator project based on Citra. It was created from the merging of PabloMK7's Citra fork and the Lime3DS project, both of which emerged shortly after Citra was taken down.
+builddeps  :
+    - pkgconfig(Qt6Multimedia)
+    - pkgconfig(RapidJSON)
+    - pkgconfig(SPIRV-Headers)
+    - pkgconfig(catch2)
+    - pkgconfig(fmt)
+    - pkgconfig(inih)
+    - pkgconfig(jack)
+    - pkgconfig(libavcodec-4.4)
+    - pkgconfig(libcryptopp)
+    - pkgconfig(libenet)
+    - pkgconfig(libpipewire-0.3)
+    - pkgconfig(libssl)
+    - pkgconfig(libusb-1.0)
+    - pkgconfig(nlohmann_json)
+    - pkgconfig(portaudio-2.0)
+    - pkgconfig(sdl2)
+    - pkgconfig(sndio)
+    - pkgconfig(soundtouch)
+    - pkgconfig(x11)
+    - pkgconfig(xext)
+    - glslang-devel
+    - libboost-devel
+    - qt6-base-private-devel
+    - robin-map
+    - vulkan-headers
+clang      : true
+optimize   :
+    - speed
+    - thin-lto
+patterns   :
+    - azahar :
```

Comment:
The key for the main package is `main`.

Could you also move the whole `patterns` thing to the bottom of the `package.yml`? Thanks!